index.html

```diff
@@ -184,5 +184,42 @@ <h3 class="format">Skills</h3>
       </li>
     </ul>
   </section>
+  <section class="form">
+    <p class="primary-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <form action="https://formspree.io/f/mwkjplad" method="post">
+      <ul class="element-form">
+        <li class="items-form">
+          <!-- <label for="name"></label> -->
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)